hypervisor/include/arch/x86/vtd.h

```diff
@@ -531,7 +531,7 @@ int unassign_iommu_device(const struct iommu_domain *domain, uint8_t bus, uint8_
  * @param[in] addr_width address width of the VM
  *
  * @return Pointer pointer to iommu_domain
- * @return NULL if translation_table is 0
+ * @retval NULL if translation_table is 0
```

Comment:
This creates two distinct sections (called "return" and "return value") in the HTML doc, but not all possible return values are listed in `retval` which is a bit confusing.

Reply:
I added another `@retval` to cover the non-NULL case. Does that look good to you?